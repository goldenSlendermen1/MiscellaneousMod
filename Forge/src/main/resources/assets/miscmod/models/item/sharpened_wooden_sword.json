{
  "parent": "item/handheld",
  "textures": {
    "layer0": "miscmod:item/sharpened_wooden_sword"
  }
}