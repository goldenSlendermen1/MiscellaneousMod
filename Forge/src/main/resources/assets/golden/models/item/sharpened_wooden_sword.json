{
  "parent": "item/handheld",
  "textures": {
    "layer0": "golden:item/sharpened_wooden_sword"
  }
}